src/views/CardStack/CardStack.js

```diff
@@ -317,12 +321,20 @@ class CardStack extends React.Component<Props> {
         const axisDistance = isVertical
           ? layout.height.__getValue()
           : layout.width.__getValue();
-        const movedDistance = gesture[isVertical ? 'dy' : 'dx'];
-        const gestureVelocity = gesture[isVertical ? 'vy' : 'vx'];
+        const movedDistance = gestureDirectionInverted
+          ? -gesture[isVertical ? 'dy' : 'dx']
```

Comment:
How about saying 
```js
const multiplier = gestureDirectionInverted ? -1 : 1;
const movedDistance = multiplier * gesture[isVertical ? 'dy' : 'dx']
```
that would imo make it easier to read

Reply:
Yeah, might be a bit clearer. Maybe the const name could be more descriptive about determining the direction of the movement, instead of the amplitude. Maybe `movementDirection`, or `directionMultiplier`?